main.py

```diff
@@ -565,3 +553,31 @@ def handle_stats_reply(self, msg, switch):
             event = KytosEvent('amlight/flow_stats.flows_updated')
             event.content['switch'] = switch.dpid
             self.controller.buffers.app.put(event)
+
+    @listen_to('kytos/of_core.flow_stats.received')
+    def handle_stats_received(self, event):
+        """Capture and handle flow stats messages for OpenFlow 1.3."""
+        msg = event.content['message']
+        switch = event.source.switch
+        if msg.multipart_type == common04.MultipartType.OFPMP_FLOW:
+            try:
+                replies_flows = event.content['replies_flows']
+                self.handle_stats_reply_received(msg, switch, replies_flows)
+            except AttributeError as err:
+                log.error(err)
+
+    def handle_stats_reply_received(self, msg, switch, replies_flows):
+        """Iterate on the replies and set the generic flows"""
+        flow = GenericFlow.from_flow_stats(replies_flows, switch.ofp_version)
```

Comment:
`GenericFlow.from_flow_stats` takes a single `FlowStats` or a `Flow` like object, you're passing a list here, this will also lead to problems. You can build a new list comprehension here and assign to `switch.generic_flows`.